darts/models/torch_forecasting_model.py

```diff
@@ -205,6 +206,12 @@ def __init__(self,
         self.lr_scheduler_cls = lr_scheduler_cls
         self.lr_scheduler_kwargs = dict() if lr_scheduler_kwargs is None else lr_scheduler_kwargs
 
+        # by default models are block models (i.e. not recurrent)
+        self.is_recurrent = False
+
+        # by default models are deterministic (i.e. not probabilistic)
+        self.is_probabilistic = False
```

Comment:
Sounds good but I think we still need an attribute, because the RNN could be probabilistic or not, so I guess we can return the attribute in the abstract method.. would that make sense?

Reply:
Update: I have now added the first version of the likelihood model abstraction (`darts.utils.likelihood_model.LikelihoodModel`), and so I am now using the `self.likelihood_model` attribute instead of `self.is_probabilistic`. Of course we could still create a `is_probabilistic()` method which returns False if `self.likelihood_model==None` and True otherwise. What do you think?